parquet-column/src/main/java/org/apache/parquet/io/api/Binary.java

```diff
@@ -148,7 +174,7 @@ public void writeTo(DataOutput out) throws IOException {
 
   private static class FromStringBinary extends ByteArrayBackedBinary {
     public FromStringBinary(byte[] value) {
-      super(value);
+      super(value, true);
```

Comment:
This isn't a huge concern because this class is private, but if this is going to statically set whether or not the binary is reused, then it should really do the conversion to a byte array in the constructor so there is no chance for the byte array to be leaked in the future.